File: VB/ReplaceQueryBuilderSample/MyConfigureQueryPageView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Data
Imports DevExpress.DataAccess.UI.Wizard.Views
Imports DevExpress.DataAccess.Wizard
Imports DevExpress.DataAccess.Wizard.Services
' ...

Namespace ReplaceQueryBuilderSample
    Friend Class MyConfigureQueryPageView
        Inherits ConfigureQueryPageView

        Public Sub New(ByVal displayNameProvider As IDisplayNameProvider, ByVal propertyGridServices As IServiceProvider, ByVal customQueryValidator As ICustomQueryValidator, ByVal options As SqlWizardOptions)
            MyBase.New(displayNameProvider, propertyGridServices, customQueryValidator, options)
        End Sub

        Public ReadOnly Property Window() As IWin32Window
            Get
                Return FindForm()
            End Get
        End Property
    End Class
End Namespace

File: CS/ReplaceQueryBuilderSample/MyConfigureQueryPageView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.Data;
using DevExpress.DataAccess.UI.Wizard.Views;
using DevExpress.DataAccess.Wizard;
using DevExpress.DataAccess.Wizard.Services;
// ...

namespace ReplaceQueryBuilderSample {
    class MyConfigureQueryPageView : ConfigureQueryPageView {
        public MyConfigureQueryPageView(IDisplayNameProvider displayNameProvider, IServiceProvider propertyGridServices, ICustomQueryValidator customQueryValidator, SqlWizardOptions options) : base(displayNameProvider, propertyGridServices, customQueryValidator, options) { }

        public IWin32Window Window { get { return FindForm(); } }
    }
}
